backend/tracim_backend/lib/webdav/resources.py

```diff
@@ -176,7 +179,14 @@ def createCollection(self, name: str):
         # TODO : remove comment here
         # raise DAVError(HTTP_FORBIDDEN)
         workspace_name = webdav_convert_file_name_to_bdd(name)
-        new_workspace = self.workspace_api.create_workspace(workspace_name)
+        try:
+            new_workspace = self.workspace_api.create_workspace(workspace_name)
+        except (
+            UserNotAllowedToCreateMoreWorkspace,
+            EmptyLabelNotAllowed,
+            WorkspaceLabelAlreadyUsed,
+        ):
+            raise DAVError(HTTP_FORBIDDEN)
```

Comment:
`DAVError` class own `contextinfo` (`str`) parameter in constructor: we can use it to add some information about error (like `DAVError(HTTP_FORBIDDEN, contextinfo=str(exc)`).

Reply:
Added this to all webdav case Forbidden error where it's possible to have explicit error.

Reply:
This is a nice feature for debugging, but those informations are not provided by webdav client like nautilus (don't know if some client does give thoses informations).